codegen/facelift/templates/ServiceIPC.template.h

```diff
@@ -280,6 +280,14 @@ class {{interface}}IPCProxy : public facelift::IPCProxy<{{interface}}PropertyAda
         int {{property.name}}Size;
         msg >> {{property.name}}Size;
         m_{{property.name}}.setSize({{property.name}}Size);
+
+        {% elif property.type.is_model %}
```

Comment:
That is redundant and models are supported now. Rebase? Same below for interface type...